rita_common/src/hello_handler/mod.rs

```diff
@@ -1,119 +1,65 @@
-//! Actor used for handling the dispatch of hello messages
+//! Function used for handling the dispatch of hello messages
 //!
 //! The call path goes like this
 //!
 //! peer listener gets udp ImHere -> TunnelManager tries to contact peer with hello
 //! -> hello manager actually manages that request -> hello manager calls back to tunnel manager
 
+use std::time::Duration;
+
 use crate::peer_listener::Peer;
+use crate::tm_identity_callback;
 use crate::tunnel_manager::id_callback::IdentityCallback;
-use crate::{tm_identity_callback, RitaCommonError};
-use actix::{Actor, Context, Handler, Message, ResponseFuture, Supervised, SystemService};
-use actix_web::client::Connection;
-use actix_web::{client, HttpMessage, Result};
+
 use althea_types::LocalIdentity;
-use futures01::future::ok as future_ok;
-use futures01::Future;
-use tokio::net::TcpStream as TokioTcpStream;
 
 #[derive(Default)]
 pub struct HelloHandler;
 
-impl Actor for HelloHandler {
-    type Context = Context<Self>;
-}
-
-impl Supervised for HelloHandler {}
-impl SystemService for HelloHandler {
-    fn service_started(&mut self, _ctx: &mut Context<Self>) {
-        info!("HelloHandler started");
-    }
-}
-
 #[derive(Debug)]
 pub struct Hello {
     pub my_id: LocalIdentity,
     pub to: Peer,
 }
 
-impl Message for Hello {
-    type Result = Result<(), RitaCommonError>;
-}
-
 /// Handler for sending hello messages, it's important that any path by which this handler
 /// may crash is handled such that ports are returned to tunnel manager, otherwise we end
 /// up with a port leak which will eventually crash the program
```

Comment:
the part of this comment talking about ports can be removed, we garbage collect now